firefox-ios/Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -856,10 +855,14 @@ class SearchViewController: SiteTableViewController,
         case .openedTabs:
             return filteredOpenedTabs.count
         case .remoteTabs:
-            return model.shouldShowSyncedTabsSuggestions ?
-            filteredRemoteClientTabs.count : 0
+            return if featureFlags.isFeatureEnabled(.firefoxSuggestFeature, checking: .buildAndUser) {
+                model.shouldShowSyncedTabsSuggestions ?
+                filteredRemoteClientTabs.count : 0
+            } else { filteredRemoteClientTabs.count }
```

Comment:
```suggestion
            } else {
                filteredRemoteClientTabs.count
            }
```
nit: a single, short & sweet `if true { doThis }` on one line looks fine. But doing this in the else block looks weird. lol